docs/guides/typescript.md

```diff
@@ -76,4 +76,135 @@ Remix has TypeScript type definitions built-in as well. The starter templates cr
 
 <docs-info>Note that the types referenced in `remix.env.d.ts` will depend on which environment you're running your app in. For example, there are different globals available in Cloudflare</docs-info>
 
+## Type Utilities
+
+Remix's built-in inference between `loader` function + `useLoaderData` generic and the `action` function + `useActionData` generic is quite good and you shouldn't often have to use thse utilities. That said they can be quite handy in some cases which is why they are exported.
```

Comment:
```suggestion
Remix's built-in inference between `loader` function + `useLoaderData` generic and the `action` function + `useActionData` generic is quite good and you shouldn't often have to use those utilities. That said they can be quite handy in some cases which is why they are exported.
```

Reply:
Thanks, but it's correct as-is.

Reply:
Hey, what does `thse` mean? Is that an abbreviation for `these` ?

Reply:
🤦‍♂️ sorry. I meant to write `these` and when I reviewed your PR I failed to see that it was missing the `e` 😅

Reply:
Fixed. Thank you very much!